src/main/java/com/connexta/search/index/Index.java

```diff
@@ -6,8 +6,14 @@
  */
 package com.connexta.search.index;
 
+import static com.connexta.search.common.configs.SolrConfiguration.CONTENTS_ATTRIBUTE_NAME;
+import static com.connexta.search.common.configs.SolrConfiguration.ID_ATTRIBUTE_NAME;
+import static com.connexta.search.common.configs.SolrConfiguration.MEDIA_TYPE_ATTRIBUTE_NAME;
+
 import com.connexta.search.common.configs.SolrConfiguration;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
+import lombok.Builder;
```

Comment:
I got a spotless failure on this file for some reason 🤔 ... Not sure if it an actual issue.

Reply:
That was supposed to have been gone. I hope I didn't fudge up the commits.